include/glow/Base/Type.h

```diff
@@ -490,7 +495,8 @@ struct Type final {
   /// \return the textual name of the element \p Ty.
   static llvm::StringRef getElementName(ElemKind Ty) {
     static const char *names[] = {
-        "float", "float16", "i8", "i16", "i32", "index32", "index64", "i8fused",
+        "float",   "float16", "i8",      "i16",  "i32",
```

Comment:
Is the spacing weird on this line? It looks like a bunch of extra spaces to me but I can't tell if that's just how it's showing up in gh

Reply:
The extra spaces are actually there -- `clang-format` aligns each element with the one below it, so it's aligning `"i16"` with `"bool"`